index.html

```diff
@@ -4,6 +4,8 @@ <h1> This is Just begining </h1>
 </head>
 <body>
   <h2> You dont have any thing to lose try your best</h2>
+  
+  <h4> Bring out the beast inside you</h4>
```

Comment:
This is really cool , I liked it can u add one more line

Reply:
Oh cool i will add it